src/main/java/frc/robot/subsystems/elevatorRoller/factory/RealElevatorRollerConstants.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.subsystems.elevatorRoller.factory;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.math.filter.Debouncer;
+import frc.robot.constants.IDs;
+import frc.robot.hardware.digitalinput.supplied.SuppliedDigitalInput;
+import frc.robot.hardware.motor.talonsrx.TalonSRXMotor;
+import frc.robot.hardware.signal.supplied.SuppliedDoubleSignal;
+import frc.robot.subsystems.elevatorRoller.ElevatorRollerStuff;
+import com.revrobotics.SparkLimitSwitch;
+
+import java.util.function.BooleanSupplier;
+
+public class RealElevatorRollerConstants {
+
+	private final static SparkLimitSwitch.Type REVERSE_LIMIT_SWITCH_TYPE = SparkLimitSwitch.Type.kNormallyOpen;
+	private final static Debouncer.DebounceType DEBOUNCE_TYPE = Debouncer.DebounceType.kBoth;
+	private final static double DEBOUNCE_TIME_SECONDS = 0.05;
+	private final static int CURRENT_LIMIT = 40;
+
+	private static void configMotor(TalonSRX motor) {
+		motor.configContinuousCurrentLimit(CURRENT_LIMIT);
+		motor.configPeakCurrentLimit(CURRENT_LIMIT);
+		motor.enableCurrentLimit(true);
+		motor.setNeutralMode(NeutralMode.Coast);
+	}
+
+	public static ElevatorRollerStuff generateRollerElevatorStuff(String logPath) {
+		TalonSRX talonSRX = new TalonSRX(IDs.TalonSRXIDs.ELEVATOR_ROLLER);
+		configMotor(talonSRX);
+
+		TalonSRXMotor talonSRXMotor = new TalonSRXMotor(logPath + "srxMotor/", talonSRX, ElevatorRollerConstants.GEAR_RATIO);
```

Comment:
Make sure gear ratio should be like this and not 1/
Some motors mult and some divide